core/templates/dev/head/pages/exploration-player-page/layout-directives/progress-nav.directive.ts

```diff
@@ -49,20 +49,22 @@ angular.module('oppia').directive('progressNav', [
         '/pages/exploration-player-page/layout-directives/' +
         'progress-nav.directive.html'),
       controller: [
-        '$scope', '$rootScope', 'PlayerPositionService', 'UrlService',
-        'PlayerTranscriptService', 'ExplorationEngineService',
-        'WindowDimensionsService', 'TWO_CARD_THRESHOLD_PX',
+        '$rootScope', '$scope', 'BrowserCheckerService',
+        'ExplorationPlayerStateService', 'PlayerPositionService',
+        'PlayerTranscriptService', 'UrlService', 'WindowDimensionsService',
         'CONTINUE_BUTTON_FOCUS_LABEL', 'INTERACTION_SPECS',
-        'ExplorationPlayerStateService',
-        function($scope, $rootScope, PlayerPositionService, UrlService,
-            PlayerTranscriptService, ExplorationEngineService,
-            WindowDimensionsService, TWO_CARD_THRESHOLD_PX,
+        'TWO_CARD_THRESHOLD_PX',
+        function($rootScope, $scope, BrowserCheckerService,
+            ExplorationPlayerStateService, PlayerPositionService,
+            PlayerTranscriptService, UrlService, WindowDimensionsService,
             CONTINUE_BUTTON_FOCUS_LABEL, INTERACTION_SPECS,
-            ExplorationPlayerStateService) {
+            TWO_CARD_THRESHOLD_PX) {
           var ctrl = this;
           var transcriptLength = 0;
           var interactionIsInline = true;
           var interactionHasNavSubmitButton = false;
+          var SHOW_SUBMIT_INTERACTIONS = [
```

Comment:
This constant seems to be for listing the interactions that have submit button only for desktop, but not for mobile? If so, maybe rename to something like `SHOW_SUBMIT_INTERACTIONS_ONLY_FOR_MOBILE`?

Reply:
Done.